draft-ietf-quic-multipath.md

```diff
@@ -1487,68 +1487,65 @@ TBD-13 (experiments use 0x004e4f5f4349445f) | NO_CID_AVAILABLE | Path abandoned
 
 # Security Considerations
 
-The multipath extension retains all the security features of {{QUIC-TRANSPORT}} and {{QUIC-TLS}}
-but requires some additional consideration regarding the following amendments:
+The multipath extension retains all security properties of {{QUIC-TRANSPORT}} and {{QUIC-TLS}}
+but requires some additional consideration regarding:
 
-- the need of potential additional resources as connection IDs are now maintained per-path;
-- the provisioning of multiple concurrent path contexts and the associated resources;
-- the possibility to create and use multiple simultaneous paths and the corresponding increased amplification risk for request forgery attacks;
-- the changes on encryption requirements due to the use of multiple packet number spaces.
+- potential additional resource usage for per-path connection IDs and multiple concurrent path contexts;
+- a potentially increased amplification risk for request forgery attacks if multiple paths are used simultaneously;
+- changes to the nonce calculation due to the use of multiple packet number spaces.
 
 
 ## Memory Allocation for Per-Path Resources
 
-The initial_max_path_id transport parameter and the Max Path ID field
-in the MAX_PATH_ID frame limit the number of paths an endpoint is willing
-to maintain and accordingly limit the associated path resources.
-
+The maximum Path ID limit in initial_max_path_id or MAX_PATH_ID frame
+limit the number of paths an endpoint is willing
+to maintain and accordingly thereby also limit the associated path resources.
```

Comment:
You can strike out either accordingly or thereby or both. Fewer words is better.